packages/frontend/src/hooks/useTokenBlacklist.js

```diff
@@ -0,0 +1,16 @@
+import { useMemo } from 'react';
+
+export function useTokenBlacklist({ tokens }) {
+    // TODO: make list dynamic, fetch from db
+    const blacklistedTokens = ['kusama-airdrop.near'];
+
+    const allowedTokens = useMemo(() => {
```

Comment:
Memo probably won't be working great here - as array will be recreated at every render.
But it should not pose huge performance issues anyway, as there is only 1 element for now.